packages/zoe/test/unitTests/contracts/test-scaledPriceAuthority.js

```diff
@@ -302,3 +292,43 @@ test('initialPrice', /** @param {ExecutionContext} t */ async t => {
   const q400 = await E(pa).quoteGiven(make(collateral.brand, 400n), debt.brand);
   t.deepEqual(q400.quoteAmount.value[0].amountOut, make(debt.brand, 14200n));
 });
+
+test.skip('types', async t => {
```

Comment:
If it's intended that this be checked in with `test.skip`, it should have a comment saying why.

Reply:
the comment on 303-305 works for me.

I guess moving it up could have made it easier to see.

Reply:
I'm still confused. I infer from the latest comment that the .ts file was intended to replace this test, but the test is still here.

I don't understand how a skipped test helps to declare types, or what other function it might serve.

Does the .ts file add code to our system, or does it only impact type declarations?